weblate/static/editor/zen.js

```diff
@@ -152,14 +152,22 @@
       focusTimeout = null;
     }
   });
-
   $document.on("focusout", ".translation-editor", function () {
     editorHasFocus = false;
     const $this = $(this);
     focusTimeout = setTimeout(() => {
       if (!editorHasFocus) {
-        // Focus lost for over 1000ms, triggering save
-        handleTranslationChange.call($this[0]);
+        const $row = $this.closest("tr");
+        const checksum = $row.find("[name=checksum]").val();
+        const statusdiv = $(`#status-${checksum}`);
+        const form = $row.find("form");
+        const payload = form.serialize();
+        const lastPayload = statusdiv.data("last-payload");
```

Comment:
This would always save on first focus out right? So if user just navigates with TAB, *all* entries would be saved? Can't it use `has-changes` class instead?